src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/Aes.cs

```diff
@@ -33,6 +35,470 @@ protected Aes()
             return (Aes?)CryptoConfig.CreateFromName(algorithmName);
         }
 
+        /// <summary>
+        ///   Computes the output length of the IETF RFC 5649 AES Key Wrap with Padding
+        ///   Algorithm for the specified plaintext length.
+        /// </summary>
+        /// <param name="plaintextLengthInBytes">
+        ///   The length of the plaintext to be wrapped, in bytes.
+        /// </param>
+        /// <returns>
+        ///   The padded length of the key wrap for the specified plaintext.
+        /// </returns>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///   <paramref name="plaintextLengthInBytes"/> is less than or equal to zero.
+        /// </exception>
+        public static int GetKeyWrapPaddedLength(int plaintextLengthInBytes)
+        {
+            if (plaintextLengthInBytes <= 0)
+            {
+                throw new ArgumentOutOfRangeException(
+                    nameof(plaintextLengthInBytes),
+                    "Plaintext length must be positive.");
```

Comment:
This should be a resource string. But, I think you can replace the whole thing with `ArgumentOutOfRangeException.ThrowIfNegative`.